ValveResourceFormat/Utils/GameFolderLocator.cs

```diff
@@ -0,0 +1,158 @@
+using Microsoft.Win32;
+using System.Globalization;
+using System.IO;
+using System.Runtime.InteropServices;
+using ValveKeyValue;
+
+namespace ValveResourceFormat.Utils
+{
+    public static class GameFolderLocator
+    {
+        public static string GetSteamPath()
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
We might as well support other OSes then: https://github.com/SteamDatabase/SteamTokenDumper/blob/c491265dd4d78f8bb0aac00320b8632c6f6a3f81/SteamClientData.cs#L200